packages-common/pluggable-widgets-generator/package.json

```diff
@@ -37,6 +37,7 @@
   "devDependencies": {
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-prettier": "^3.1.2"
+    "eslint-plugin-prettier": "^3.1.2",
+    "prettier": "^1.19.1"
```

Comment:
why do you need it? I think dependency on the top of the repository should be enough

Reply:
I added it, because it's a peer dependency of eslint plugin. I tried solving a npm link issue, which if I think about it now wasn't necessary at all. It works with just in the root as well. Is this the default approach to such things? It might be handy to have it in there when we remove it from the root sometime...